.env.example

```
@@ -0,0 +1,7 @@
+POSTGRES_DB=agent_board
+POSTGRES_USER=agent_board
+POSTGRES_PASSWORD=agent_board
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)